.vsts-ci/mac.yml

```diff
@@ -55,7 +55,8 @@ jobs:
   - template: ../tools/releaseBuild/azureDevOps/templates/insert-nuget-config-azfeed.yml
 
   - powershell: |
-      tools/travis.ps1 -Stage Bootstrap
+      Import-Module .\tools\ci.psm1
+      Invoke-LinuxTests
```

Comment:
I don't see where you added `Invoke-LinuxTests`.

But, this looks better.

Reply:
I added it in the PSM1 but for some reason the CI says not found instead of running tests.